openassessment/xblock/test/data/update_xblock.json

```diff
@@ -39,6 +39,9 @@
         "allow_multiple_files": true,
         "show_rubric_during_response": false,
         "allow_latex": false,
+        "allow_learner_resubmissions": true,
+        "resubmissions_grace_period_hours": 0,
+        "resubmissions_grace_period_minutes": 0,
```

Comment:
Question: why are we always using the same values for hours and minutes? could we add different ones?

Reply:
They are at 0 because it is the default, but it is possible to use any other value.